cfgov/jinja2/v1/browse-basic/index.html

```diff
@@ -86,98 +117,14 @@ <h2 class="header-slug">
                             </span>
                         </h2>
                     {% elif 'paragraph' in block.block_type %}
-                        {{ render_stream_child(block) }}
+                        <p>{{ render_stream_child(block) }}</p>
```

Comment:
Could be removed for simplicity sake, but I'll also fix the TODO so that we know to remove it when the atoms go away.